### PR TITLE
ESS: 15.09-2 (new formula)

### PR DESCRIPTION
**Pull request by simonbyrne**
Add Emacs Speaks Statistics, an add-on package for emacs for interaction with R and other statistical packages.

### CODE REVIEW

File: ess.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  desc "Emacs Speaks Statistics"
+  homepage "http://ess.r-project.org/"
+  url "http://ess.r-project.org/downloads/ess/ess-15.09-2.tgz"
+  version "15.09-2"
+  sha256 "706c41237e1edf33a369902f503bb25254b2bbb750b9ed1adee244e875264afb"
+
+  depends_on :emacs => "23"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}", "EMACS=#{which "emacs"}"
```
Comment (dpo) [inline ess.rb]: The `:emacs` dependency sets `ENV["EMACS"]`. Is that not sufficient here?

File: ess.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  end
+
+  def caveats
+    <<-EOS.undent
```
Comment (dpo) [inline ess.rb]: Most formulas combine lines 14 and 15. We should stick with the most common style.

File: ess.rb (hunk 1 above)
Comment (simonbyrne) [inline ess.rb]: Ah, I didn't know that, I was basing it on the [auctex one](https://github.com/Homebrew/homebrew-tex/blob/95905f3106773e90955503f000fe27b3915dd450/Formula/auctex.rb#L23). Will fix it.

File: ess.rb (hunk 2 above)
Comment (simonbyrne) [inline ess.rb]: will do.

File: ess.rb (hunk 1 above)
Comment (simonbyrne) [inline ess.rb]: Actually, I don't know: by default the Makefile set `EMACS=emacs`, will the environmental variable override that? Or do I need to do something else?

Comment (dpo) [inline ess.rb]: I was thinking `EMACS=#{ENV["EMACS"]}`

**Comment by simonbyrne** [comment]
@dpo Thanks, I've updated the formula.

**Comment by dpo** [comment]
Great! Thank you!

**Pull request closed without merging**